draft-ietf-dnsop-nsec-aggressiveuse.xml

```diff
@@ -185,8 +185,9 @@
       providing a (DNSSEC secured) record containing the names which appear
       alphabetically before and after the queried for name. In the example
       above, if the (DNSSEC validating) recursive server were to query for
-      lion.example.com it would receive a (signed) NSEC/NSEC3 record stating
-      that there are no labels between "elephant" and "zebra". This is a
```

Comment:
Thank you. We'd tried working something like this, but it kept getting really long and confusing. Your text is simple, clear and effective.

Thanks.